iptables.go

```diff
@@ -36,6 +36,13 @@ func newIPTables(config *bouncerConfig) (interface{}, error) {
 		CheckIptableCmds: [][]string{},
 	}
 
+	var target string
+  if strings.EqualFold(config.DenyAction, "REJECT") {
```

Comment:
Do we want to use `strings.Upper(config.DenyAction)` to improve user experience? (so upper and lower case word will be accepted)